### PR TITLE
BOAC-2193: Autocompletes topic with match from suggested topics

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.ets.berkeley.edu/jira/browse/BOAC-2193

I tried to make this as unobtrusive as possible. It would be great to have others test it and report their annoyance levels.

### CODE REVIEW

File: src/components/note/AdvisingNoteTopics.vue [hunk 1]
```diff
@@ -75,12 +78,36 @@ export default {
...
+  },
   methods: {
     addTopic() {
       if (this.topic && this.topic.trim()) {
         this.functionAdd(this.topic);
       }
       this.topic = undefined;
+    },
+    normalizeString(str) {
+      return str.toUpperCase().replace(/[^A-Za]/g, '');
```
Comment (paulkerschen) [inline src/components/note/AdvisingNoteTopics.vue]: @lyttam Looks like this regex would preserve lowercase `a` along with uppercase letters - typo or quirk of the topics?

Comment (lyttam) [reply src/components/note/AdvisingNoteTopics.vue]: @paulkerschen thank you, that is a typo. Will amend.

Comment (lyttam) [reply src/components/note/AdvisingNoteTopics.vue]: @paulkerschen all fixed.

**Comment by paulkerschen** [comment]
Let's give it a spin!

**Pull request merged**